### DISCUSSION TITLE
Enhancement Request SDM630

### DISCUSSION THREAD

**Discussion by Jambola77** [arendst/Tasmota#13893] [Q&A] [answered]
Hello,
it would be great to add the total energy import and export values to the file xnrg_10_sdm630.ino and maybe others.
Values are:
0x0048, // SDM630 Import
0x004A, // SDM630 Export
Kind regards,
Pete

**Comment by spcqike** [comment, marked as answer]
you may have a look at smart meter interface
https://tasmota.github.io/docs/Smart-Meter-Interface/#sdm530-modbus

just add, whatever reading you want.
